src/components/v2/EnableToken/index.tsx

```diff
@@ -1,20 +1,20 @@
 /** @jsxImportSource @emotion/react */
 import React, { useContext } from 'react';
-import BigNumber from 'bignumber.js';
 import Typography from '@mui/material/Typography';
 import { AuthContext } from 'context/AuthContext';
 import { TokenId } from 'types';
-import { formatApy } from 'utilities/common';
 import useApproveToken from 'clients/api/mutations/useApproveToken';
 import { Icon, IconName } from '../Icon';
 import { PrimaryButton } from '../Button';
 import useStyles from './styles';
+import { Delimiter } from '../Delimiter';
+import InfoRow, { IInfoRowProps } from '../InfoRow';
 
 export interface IEnableTokenProps {
   symbol: TokenId;
   isEnabled: boolean;
-  title: string;
-  tokenInfo: { symbol: TokenId; text: string; apy: BigNumber | number }[];
+  title: string | React.ReactElement;
+  tokenInfo: IInfoRowProps['info'];
   approveToken: () => void;
   vtokenAddress: string;
   disabled: boolean;
```

Comment:
I don't mind changing this but I think it is 100% acceptable as written. It would be easy to make this optional when we have an implementation that doesn't use it which we currently don't.